petab/problem.py

```diff
@@ -104,6 +104,31 @@ def __setattr__(self, name, value):
         else:
             super().__setattr__(name, value)
 
+    def __str__(self):
+        model = f"with model ({self.model})" if self.model else "without model"
```

Comment:
`Model` has no `__str__`, what gets printed here?

Reply:
Currently the not so informative default __repr__. The plan would be to add something more meaningful there as well.